README.md

```diff
@@ -0,0 +1,91 @@
+# Map colonies - priority queue management package
+<br />
+
+
+ ## Overview
+ Pyority-Queue - python priority queue package provides ability to manage queue by handle tasks statuses and report tasks liveness.
+
+ <br />
+
+ ##  Installation:
+
+ <br />
+
+From PyPi
+
+```
+ $ pip install mc-pyority-queue
+ ```
+
+<br />
+<br />
+
+ 
+## Getting Started
+
+<br />
+
+*Import & Initialize*
+
+
+
+```
+from mc-pyority-queue.task_handler import TaskHandler
+
+task_handler = TaskHandler('job_type', 'task_type',
+    'http://localhost:8081', 'http://localhost:8080/heartbeat', 1.0)
```

Comment:
why 1.0? do you support 0.5?

Reply:
yes, it is support 0.5 or 0.001, 01, etc..